code/online_challenges/src/project_euler/problem_012/README.md

```diff
@@ -0,0 +1,31 @@
+# Project Euler Problem #012: Highly divisible triangular number
+
+([Problem Link](https://projecteuler.net/problem=12))
+
+
+
+The sequence of triangle numbers is generated by adding the natural numbers. So the 7th triangle number would be 1 + 2 + 3 + 4 + 5 + 6 + 7 = 28. The first ten terms would be:
+
+1, 3, 6, 10, 15, 21, 28, 36, 45, 55, ...
+
+Let us list the factors of the first seven triangle numbers:
+
+     1: 1
+     3: 1,3
+     6: 1,2,3,6
+    10: 1,2,5,10
+    15: 1,3,5,15
+    21: 1,3,7,21
+    28: 1,2,4,7,14,28
+
+We can see that 28 is the first triangle number to have over five divisors.
+
+What is the value of the first triangle number to have over five hundred divisors?
```

Comment:
Please read [documentation guide](https://github.com/OpenGenus/cosmos/blob/master/guides/documentation_guide.md)

Reply:
Please help me with this.

Reply:
Add a title 'Explanation' from col:2 to col:23.

Reply:
@iattempt The documentation guide does _NOT_ apply here. We have our own template: https://github.com/OpenGenus/cosmos/blob/master/code/online_challenges/src/project_euler/README.md#readme-template

If you want an explanation header, please first update the template _before_ suggesting the changes. Otherwise, we end up with a lot of inconsistencies.

Reply:
Yes, my bad. Just ignore this.